package.json

```diff
@@ -5,12 +5,12 @@
     "doc": "docs"
   },
   "engines": {
-    "node": "20.9.0",
-    "pnpm": "8.10.0"
+    "node": "^20.9.0",
+    "pnpm": "^8.10.0"
   },
   "volta": {
-    "node": "20.9.0",
-    "pnpm": "8.10.0"
+    "node": "^20.9.0",
```

Comment:
> Volta v1.1.1
> 
> Could not parse version "^20.9.0"
> 
> Please verify the intended version.
> 
> Error cause: encountered unexpected token: Caret

Reply:
Ugh was mixing it up with engines, good point I will change this back later.

Reply:
Fixed in https://github.com/ferdium/ferdium-recipes/pull/481